integrated_channels/integrated_channel/apps.py

```diff
@@ -1,7 +1,7 @@
 """
 Enterprise Integrated Channel Django application initialization.
 """
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```

Comment:
I added this because of one of the linters - after I enabled more of them that I had forgotten before this came up for a bunch of files. I can look into why it is that way.